Code/Tools/AssetProcessor/native/ui/MainWindow.cpp

```diff
@@ -403,14 +404,55 @@ void MainWindow::Activate()
 
     m_jobsModel->PopulateJobsFromDatabase();
 
+    // Builders Tab:
+
+    ui->builderList->setModel(&m_builderList);
+    connect(ui->builderList->selectionModel(), &QItemSelectionModel::selectionChanged, [this](const QItemSelection& selected, const QItemSelection& /*deselected*/)
```

Comment:
MainWindow::Activate is already a very large function, can you maybe pull this lambda out?